### ISSUE TITLE
weather wind speed shows incorrect units

### ISSUE THREAD

**Issue by lanrat** [DavidFW1960/bom-weather-card#62]
The [weather](https://www.home-assistant.io/integrations/weather/) integrations for Home Assistant supports a `wind_speed` attribute. However no unit seems to be defined.

In my case, I'm using the default [Meteorologisk institutt (Met.no)](https://www.home-assistant.io/integrations/met/) integration, which provides the wind-speed in km/h, however this card displays the data as mi/h. Home Assistant does not do any unit conversions, which results in the data being displayed with the wrong unit.

I assume that this is because I have Home Assistant set to use imperial units, but this is resulting in incorrect data being displayed. To fix this could this card be updated to  display the wind speed in the weather card as km/h instead of mi/h IF the source returns data in km/h?
Alternatively this card can do km/h -> mi/h conversion, but I think that may be out of scope.

Related issue filed in Home Assistant core: https://github.com/home-assistant/core/issues/55824

**Comment by DavidFW1960** [comment]
Yes the card uses the locale parsed to the card via the locale flag or it will use the locale in the browser so if HA is configured as imperial.
I am certain you will find the source is not returning ANY unit for the windspeed and you can probably see that in the state for the entity you are parsing.
see here:
![image](https://user-images.githubusercontent.com/36390762/136122068-f1f989fb-ebda-4b43-b7ea-a160c02e1027.png)
As you can see no units are in the entity..
What you can do is make a template sensor to convert the km/h windspeed from met.no to mi/h and then parse that entity to the card instead of the km/h from the weather sensor.

**Comment by lanrat** [comment]
A template sensor is one option, although that feels like overkill.

Would it be possible to manually specify the unit for the bom-weather-card for all entities that don't have a unit attached? This would allow setting the unit to whatever the source is so that it displays the information accurately.

**Comment by DavidFW1960** [comment]
and you think using a template for the one entity that is bothering you is overkill?
I'm not going to change the card for that. The correct way to specify a unit for an entity is via the locale which is what I do. Note also that the 'source' isn't specifying any unit at all - for ANY entity. They are ALL unitless.

**Comment by DavidFW1960** [comment]
I don't believe so. It could be included in the state (some integrations do this. This is controlled by the integration,
